CHANGELOG.md

```diff
@@ -1,3 +1,174 @@
+# 0.28.0 (2019-04-??)
+
+## Language changes
+
+- **(breaking-change)** Enum declarations members need to be separated by a new-line (or `;`). ([#7607](https://github.com/crystal-lang/crystal/pull/7607), thanks @asterite)
+- Add begin-less and end-less ranges. ([#7179](https://github.com/crystal-lang/crystal/pull/7179), thanks @asterite)
+- Add `offsetof(Type, @ivar)` expressions. ([#7589](https://github.com/crystal-lang/crystal/pull/7589), thanks @malte-v)
+
+### Macros
+
+- Add `Type#annotations` to list all annotations and not just the last of each kind. ([#7326](https://github.com/crystal-lang/crystal/pull/7326), thanks @Blacksmoke16)
+- Add `ArrayLiteral#sort_by` macro method. ([#3947](https://github.com/crystal-lang/crystal/pull/3947), thanks @jreinert)
+
+## Standard library
+
+- **(breaking-change)** Allow creating `None` enum flag with `Enum.from_value`. ([#6516](https://github.com/crystal-lang/crystal/pull/6516), thanks @bew)
+- **(performance)** Optimize dwarf line numbers decoding. ([#7413](https://github.com/crystal-lang/crystal/pull/7413), thanks @asterite)
+- Fix `Signal::CHLD.reset` enot clearing previousl handler. ([#7409](https://github.com/crystal-lang/crystal/pull/7409), thanks @asterite)
+- Add lazy versions of `Object#getter?` and `Object#property?` macros. ([#7322](https://github.com/crystal-lang/crystal/pull/7322), thanks @Sija)
+- Allow returning other values than `-1`, `0` and `1` by `Comparable#<=>` and `PartialComparable#<=>`. ([#7277](https://github.com/crystal-lang/crystal/pull/7277), thanks @r00ster91)
```

Comment:
I wouldn't mention `PartialComparable` here. It shouldn't be used anyway.

Reply:
I mean it should be noted as a breaking change. I don't think it should just magically disappear. That can be very confusing.